src/db/migrate.rs

```diff
@@ -74,12 +74,6 @@ pub fn migrate(version: Option<Version>, conn: &mut Client) -> CratesfyiResult<(
                  latest_version_id INT DEFAULT 0,
                  versions JSON DEFAULT '[]',
                  downloads_total INT DEFAULT 0,
-                 github_description VARCHAR(1024),
-                 github_stars INT DEFAULT 0,
-                 github_forks INT DEFAULT 0,
-                 github_issues INT DEFAULT 0,
-                 github_last_commit TIMESTAMP,
-                 github_last_update TIMESTAMP,
```

Comment:
These columns still need to exist for later migrations to modify, until migration 23 is run.

Reply:
Removed that migration, will put it as its own separate PR later.